daemon/cmd/cni/cell.go

```diff
@@ -62,6 +64,14 @@ func enableConfigManager(lc hive.Lifecycle, log logrus.FieldLogger, cfg Config,
 }
 
 func newConfigManager(log logrus.FieldLogger, cfg Config, debug bool) *cniConfigManager {
+	if cfg.CNIChainingMode == "aws-cni" {
```

Comment:
How about `if cfg.CNIChainingMode == "aws-cni" && cfg.CNIChainingTarget == ""` ? This gives us some flexibility in case the name is changed.

Reply:
nevertheless, good point. :+1: